sdks/java/io/jdbc/src/test/java/org/apache/beam/sdk/io/jdbc/JdbcUtilTest.java

```diff
@@ -28,6 +44,66 @@
 @RunWith(JUnit4.class)
 public class JdbcUtilTest {
 
+  static final JdbcReadWithPartitionsHelper<String> PROTOTYPE_STRING_PARTITIONER =
```

Comment:
Are you planning to park the code here? Would suggest to at least add a comment with the follow up ticket

Reply:
I've added a comment referencing the issue. LMK if it looks good. Thanks so much!

Reply:
👍